test/docfx.Test/ModuleInitializer.cs

```diff
@@ -18,8 +18,8 @@ public static void Initialize()
         {
             Environment.SetEnvironmentVariable("DOCFX_APPDATA_PATH", Path.GetFullPath("appdata"));
             Environment.SetEnvironmentVariable("DOCFX_HOST_NAME", "docs.com");
-            Environment.SetEnvironmentVariable("DOCFX_OUTPUT__JSON", "true");
-            Environment.SetEnvironmentVariable("DOCFX_OUTPUT__COPY_RESOURCES", "true");
+            Environment.SetEnvironmentVariable("DOCFX_OUTPUT_JSON", "true");
```

Comment:
since there is no nested type in config, can we remove `objectSeparator` from `StringUtility.ExpandVariables`?